### ISSUE TITLE
Support for React Native 0.76.0

### ISSUE THREAD

**Issue by marcin-krzyzanowski** [portabletext/react-native-portabletext#165]
Using `portableText@1.1.13` with React Native **0.76.0** results such an issue:

![Screenshot 2024-11-25 at 11 29 34 AM](https://github.com/user-attachments/assets/6b80cdd1-74a6-4607-9ed4-39f4a45b0821)

I guess it may come from that the current version `portabletext` relies on react-native@0.74.5 and then causing the whole app gets mismatched react-native versions. (My app is using react native 0.76.0)

**Comment by Jojocaster** [comment]
Yeah that'd be great, we're stuck on RN 0.74 because of PortableText.

**Comment by guidolarosa** [comment]
Hi! I think it's awesome you made this package, but seems like using it with the latest version of react-native (0.76.0) makes the app crash. Would love to see that the PR that @Jojocaster is talking about would fix the issue.

Thanks!

**Comment by gfirmill** [comment]
Any update on this?

**Comment by csillaj** [comment]
I'm also blocked in updating to RN 0.76.0 by this package

**Comment by qpavy** [comment]
This will began to be mandatory to update dependencies to use RN 0.76 as expo SDK 52 is forcing the use of RN 0.76. Soon we won't be able to update our app to the stores

**Comment by luckywacky** [comment]
still nothing? expo 53 is here already and we can't use this library even on expo 52  ;(
